proposals/0111-linear-types.rst

```diff
@@ -265,45 +265,46 @@ Syntax
 This proposal adds two new syntactical constructs:
 
 - The multiplicity annotated arrow, for polymorphism, is written ``a
-  #p -> b`` (where ``a`` and ``b`` are types and ``p`` is a
-  multiplicity). This steals syntax as ``(#)`` is a valid
-  type operator. That is the syntax entry for types becomes:
+  ^p -> b`` (where ``a`` and ``b`` are types and ``p`` is
+  a multiplicity). We add a new production to the grammar for
+  ``type``:
 
   ::
 
-    type -> btype [[# btype] -> type]
+    btype PREFIX_CARET btype -> ctype
```

Comment:
That should be `atype` after the `PREFIX_CARET` (or `mult`, which, in turn, should be defined as an `atype`).